cypress/e2e/home.cy.ts

```diff
@@ -1,9 +1,9 @@
-context('Basic', () => {
+context('Home', () => {
   beforeEach(() => {
     cy.visit('/')
   })
 
-  it('basic nav', () => {
+  it('go to test page', () => {
```

Comment:
Wouldn't it will be better to write `home` page here?
or we could rename `Home` to test to keep the consistency of our Behavioral tests